### PR TITLE
Unify frontend

### PR DESCRIPTION
**Pull request by wes-curl**
Added linkages between pages. Also performed various small fixes. Frontend now fully linked to backend.

### CODE REVIEW

File: frontend/src/App.js
Comment (sajanjohal) [inline frontend/src/App.js:13]: What does useState("nothin'") do? Can you change it to look slightly more professional?

**Review by sajanjohal — COMMENTED** [review]
Did you run prettier on this code? If not, can you?
Also, there seem to be some commits in here that are similar to ones made earlier on the backend. Not sure if this will be a problem but it could cause some confusing conflicts down the line.

**Comment by sleepyStick** [comment]
if you can resolve the conflicts, that'd be great :) 
otherwise, ship it 👍

**Review by sleepyStick — CHANGES_REQUESTED** [review]
if you can resolve the conflicts, that'd be great :)
otherwise, ship it 👍

File: frontend/src/App.js
Comment (wes-curl) [reply frontend/src/App.js:13]: Oop! I meant to use that as a testing value. I will change that.

**Comment by wes-curl** [comment]
I did not. I will run it when the PR is closed, so I don't have to wait for the entire team's approval to do it again. Those backend commits should have no effect, by the way. I did another review and there doesn't seem to be any risk of conflict.

**Review by sleepyStick — APPROVED** [review]

**Review by ashahrod — APPROVED** [review]
Looks good!

**Review by aeller-design — APPROVED** [review]
looks good to me!

**Comment by aeller-design** [comment]
great job, looks good to me

**Review by sajanjohal — APPROVED** [review]
Looks good!

**Pull request merged**
